modules/setting/setting.go

```diff
@@ -90,6 +90,7 @@ var (
 	SSH = struct {
 		Disabled             bool           `ini:"DISABLE_SSH"`
 		StartBuiltinServer   bool           `ini:"START_SSH_SERVER"`
+		BuiltinSSHServerUser string         `ini:"BUILTIN_SSH_SERVER_USER"`
```

Comment:
Idea: What about changing `BuiltinSSHServerUser` to `BuiltinServerUser`. Then reading config inside go `setting.SSH.BuiltinServerUser`. Also "builtin" word can be removed so it would be more consistent with other ssh settings. What do you think?

Reply:
I'll remove the SSH from the var name. Since this user only applies to the internal server, I'll keep the Builtin

Reply:
Good point. The last thing: `SSH_BUILTIN_SERVER_USER` instead of `BUILTIN_SSH_SERVER_USER`?